bitnami/cilium/templates/operator/vpa.yaml

```diff
@@ -0,0 +1,45 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if and (.Capabilities.APIVersions.Has "autoscaling.k8s.io/v1/VerticalPodAutoscaler") .Values.operator.autoscaling.vpa.enabled }}
```

Comment:
Don't we want a vertical autoscaler in the agent daemonset? HPA would not make sense but it could make sense to vertically increase its resources

Reply:
done at https://github.com/bitnami/charts/pull/26374/commits/c5ee18dd06171ea3c925861c0811632c8cf7281c